genesis-blocks/docker-compose.host.docker.internal.yml

```diff
@@ -0,0 +1,55 @@
+version: '3.5'
```

Comment:
The new `proposer` and `genesis-blocks-deployer` will be part of 
- docker-compose.yml
- docker-compose.images.example.yml
- docker-compose.dev.yml

This genesis blocks deployment is needed for all our current deployments.

So we don't need new docker-compose files in genesis-blocks.